review_checker/views/android_events.view.lkml

```diff
@@ -0,0 +1,127 @@
+include: "//looker-hub/review_checker/views/android_events.view.lkml"
+
+
+view: +android_events {
+
+  dimension: client_id {
+    #primary_key: yes
+    sql: ${TABLE}.client_id ;;
+    hidden: yes
+  }
+
+  dimension: client_id_date {
+    primary_key: yes
+    sql: CONCAT(${TABLE}.client_id, ${TABLE}.submission_date) ;;
+    type: string
+  }
+
+  dimension: is_opt_in {
+    type: number
+    sql: ${android_clients.is_opt_in} ;;
```

Comment:
I'm confused about how this part works, but I'll assume it's working properly given you were able to commit

Reply:
here i'm just pulling in the is_opt_in dimension to make it accessible in the events explore.